samples/drivers/sdhc/sdhc_read_write/boards/arduino_nicla_vision_stm32h747xx_m7.conf

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2025 EXALT Technologies.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SDMMC_STM32=y
+CONFIG_CYW4343W=y
+CONFIG_CYW4343W_MURATA_1DX=y
+CONFIG_LOG_MODE_DEFERRED=y
```

Comment:
Most of those are likely not required as indirectly enabled by the flags enabled in prj.conf.
Please check for a possible clean up.